flutter-idea/src/io/flutter/sdk/FlutterSdkVersion.java

```diff
@@ -59,7 +59,7 @@
   @NotNull
   private static final FlutterSdkVersion MIN_PASS_DEVTOOLS_SDK = new FlutterSdkVersion("1.26.0-11.0.pre");
   @NotNull
-  private static final FlutterSdkVersion MIN_OPTIONAL_PASS_DEVTOOLS_SDK = new FlutterSdkVersion("2.7.0-3.0.pre");
+  private static final FlutterSdkVersion MIN_OPTIONAL_PASS_DEVTOOLS_SDK = new FlutterSdkVersion("2.6.0-12.0.pre.854");
```

Comment:
what version of DevTools is included with the flutter version? In DevTools, we will show a warning to users to stop launching from pub if they are on Flutter 2.8.0 or Dart 2.15 or greater. The reasoning behind this is we didn't want to downgrade users by telling them to stop launching from pub. My concern with using Flutter 2.6.0-12.0.pre.854 is that we may downgrade a user's devtools version by doing this. Maybe we should gate instead on flutter 2.8.0, which we know has DevTools 2.8.0 included.

Reply:
`dart devtools` about box shows 2.8 for me. I don't know much about devtools -- all I did was update my master flutter branch to latest and got that.